.github/workflows/lint.yml

```diff
@@ -0,0 +1,56 @@
+name: Lint tests
+
+on:
+  push:
+    branches-ignore:
+    - 'release/*'
+  pull_request:
+env:
+  CORE_REPO_SHA: 955c92e91b5cd4bcfb43c39efcef086b040471d2
+
+jobs:
+  lint-3_11:
```

Comment:
I think we are on python [3.12](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/.github/workflows/lint.yml#L12) now

Reply:
Right, fixed